components/Ripple/src/MDCRippleView.m

```diff
@@ -57,6 +57,7 @@ - (instancetype)initWithCoder:(NSCoder *)aDecoder {
 }
 
 - (void)commonMDCRippleViewInit {
+  self.usesSuperviewShadowLayerAsMask = YES;
```

Comment:
Nit: Shouldn't we be using the `ivar` here?

Reply:
Fixed.